### ISSUE TITLE
How to use this module mimo?

### ISSUE THREAD

**Issue by Amberlan1001** [zhouyizirui/Wireless#2]
Dear author：
   I am a rookie in gnuradio, would you exactly tell me how to use those module? when I run it ,there is a error,which said no module named mimo. Thanks for your  help.

**Comment by zhouyizirui** [comment]
Hi there,

Thanks for your attention. This project was done nearly two years ago, when I was in my graduate study. Unfortunately, the project here is incomplete, some modules and blocks are lost(I can't find them neither now).

As for building MIMO in GNU Radio, I think there are two aspects. First you may need to design and implement C++ blocks to do the digital signals processing, including synchronization, encoding, decoding and so on. Second you should write Python scripts to connect the blocks and build the flowgraph. If you want to do real-world experiments, some cables will be needed to connect USRPs together.

If you face technical problems, the GNU Radio official website and mailing-list may be helpful. 

Anyway, welcome to contribute to this repository in case others face the same problem.

**Comment by Amberlan1001** [comment]
It is very nice to receive your reply. I have run successfully in USRP, and in the website of 
https://www.ruby-forum.com/topic/4423225, you said "MIMO node can utilize Beamforming techniques to precode The symbols transmitted by the two antennas."  Is that  Python scripts of 
Beamforming techniques in this project have lost？which need I write blocks to implement? Thanks.

**Comment by zhouyizirui** [comment]
Yes, unfortunately the Beamforming codes are lost. You may want to write C++ blocks to implement precoding and connect them with Python scripts. Remember that synchronisation is important in MIMO, you may also want to use MIMO cables in USRP.
